byoqm/visuals/dashboard.py

```diff
@@ -0,0 +1,39 @@
+import csv
+import os
+
+from bokeh.layouts import gridplot
+from bokeh.plotting import show
+from visuals import line
+
+
+class Dashboard:
+    def show_graphs(self):
+        data = self.get_data()
+        f = [line.get_line(data, key) for key in data]
+        grid = gridplot([[f[0], f[1]], [f[2], f[3]], [f[4], f[5]]])
+        show(grid)
+
+    def sort_data_values(self, data):
+        for key in data:
+            data[key].sort()
```

Comment:
Just to understand better, why do we need to sort?

Reply:
![image](https://user-images.githubusercontent.com/60236750/221150460-18d49898-05a1-4b42-8ab9-7e931ad6f95c.png)
to avoid this